src/AutoRest.CSharp/Mgmt/Generation/ResourceContainerWriter.cs

```diff
@@ -226,6 +228,21 @@ private void WriteContainerMethodScope(bool isAsync, FormattableString returnTyp
             }, catch404);
         }
 
+        private void WriteGetFromRestClient(RestClientMethod method, bool async)
+        {
+            var parameterMapping = BuildParameterMapping(method);
+            string await = async ? "await " : string.Empty;
+            string asyncSuffix = async ? "Async" : string.Empty;
+            string configAwait = async ? ".ConfigureAwait(false)" : string.Empty;
+            _writer.Append($"var response = {await}{RestClientField}.{method.Name}{asyncSuffix}(");
+            foreach (var parameter in parameterMapping)
+            {
+                _writer.AppendRaw(parameter.IsPassThru ? parameter.Parameter.Name : parameter.ValueExpression);
+                _writer.AppendRaw(", ");
+            }
```

Comment:
We have a function called `WriteArguments(parameterMapping)` which does the same thing as this loop

Reply:
Or use `BuildAndWriteParameters()` and remove `var parameterMapping = BuildParameterMapping(method);`?